<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>482</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>送出</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>611</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <family>微軟正黑體 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>LINE：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>微軟正黑體 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>金額：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>微軟正黑體 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Line">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_Amount">
       <property name="font">
        <font>
         <family>微軟正黑體 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Name">
       <property name="font">
        <font>
         <family>微軟正黑體 Light</family>
         <pointsize>14</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <family>微軟正黑體 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>電話：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_Phone">
       <property name="font">
        <font>
         <family>微軟正黑體</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBox_location">
       <property name="font">
        <font>
         <family>微軟正黑體 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <item>
        <property name="text">
         <string>台北基隆</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>桃竹苗</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>彰中投</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>雲嘉南</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>高屏</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <family>微軟正黑體 Light</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>區域：</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
